src/runtime-rs/crates/runtimes/virt_container/src/lib.rs

```diff
@@ -149,4 +161,29 @@ agent_name="kata"
         let res = new_agent(&toml_config);
         assert!(res.is_ok());
     }
+
+    #[test]
+    fn test_new_hypervisor() {
+        VirtContainer::init().unwrap();
+
+        let toml_config = {
+            let config_content = r#"
+[hypervisor.qemu]
+path = "/bin/echo"
+kernel = "/bin/echo"
+image = "/bin/echo"
+firmware = ""
+
+[runtime]
+hypervisor_name="qemu"
+            "#;
```

Comment:
This line contains only spaces, so can delete the spaces.

Reply:
Fixed (I viewed the formatting from the source code point of view rather than the contents of the "file" so I placed it like a closing bracket but a fair point).